app/views/surveys/show.html.erb

```diff
@@ -1,4 +1,37 @@
 <h1> <%= @survey.name %> </h1>
-<%= link_to 'Edit Survey', edit_survey_path(@survey) %>
-<%= link_to 'Delete', @survey, method: :delete, data: { confirm: 'Are you sure?' } %>
-<%= link_to 'Fill it', new_survey_user_response_path(@survey) %>
\ No newline at end of file
+
+<div class = "jumbotron">
```

Comment:
this div is being used in js? if this div tag is being used, then add a comment what will be added here